### ISSUE TITLE
Better reporting

### ISSUE THREAD

**Issue by sumobi** [AffiliateWP/AffiliateWP#616]
Clearly there are areas where we could improve reporting. For now, let's just make a list of some ideas:

A customer has expressed interest in seeing a summary of affiliate stats which clearly outline the affiliates, how many affiliate sales they have generated, amount, and a list of referrals. 

<blockquote>
With 689 sales from the one affiliate, I have to go through many many pages clicking next, next, next until I find a different name. Is there any way at all to simply see a list of the affiliates with sales, their total referrals and amount owed total?
</blockquote>

While you can get some of this data from the referrals page, you can't get an overview of everything at once. Perhaps something like:

Affiliate 1 | 23 sales, $400, referrals (drill down to referral list)
Affiliate 2 | 36 sales, $874, referrals (drill down to referral list)
Affiliate 3 | 5 sales, $40, referrals (drill down to referral list)

Everything should exportable. Pippin and I have so far discussed 1 reporting area where everything is able to be filtered with the data and graphs updating in real time.

**Comment by sumobi** [comment]
Some feedback from a customer:

> Affiliates need to be able to access a custom range of time period and be able to find out what was their paid and unpaid earnings/sales for that period. The graphs show that data but there is no way to add it up. Today if an affiliate wanted to know exactly how much he earned in July 2015, how would he do that?

**Comment by pippinsplugins** [comment]
:+1:

**Comment by pippinsplugins** [comment]
Unfortunately this isn't going to make 1.7.

**Comment by worldtravlr** [comment]
In a previous affiliate plugin I used there was an additional sub menu option in the sidebar for Totals.  Total would show Affiliate ID, Affiliate Name, Affiliate Email, Affiliate Username and Total Commissions.  This could be sorted by any of these criteria, but the most intuitive and helpful to me was by Total as I could go in and see who my top affiliates were very quickly and easily even if they had not been paid yet and also see how much they were due to be paid and/had been paid in their time as affiliates.

I know AffiliateWP has something maybe kind of similar in the Overview under Most Valuable Affiliates, but it does not show all affiliates and either mine is not functioning or it only shows based on paid referrals and does not include unpaid referrals.  This would be a nice feature to have as well I believe.
